channels/ca.m3u

```diff
@@ -139,8 +139,8 @@ http://cdn12.henico.net:8080/live/gsctv/index.m3u8
 https://ici-i.akamaihd.net/hls/live/873426/ICI-Live-Stream/master.m3u8
 #EXTINF:-1 tvg-id="IciRDI.ca" tvg-country="CA" tvg-language="French" tvg-logo="https://raw.githubusercontent.com/geonsey/Free2ViewTV/master/images/logos/ICI-RDI.png" group-title="News",Ici RDI (720p)
 https://rcavlive.akamaized.net/hls/live/704025/xcanrdi/master.m3u8
-#EXTINF:-1 tvg-id="ICITele.ca" tvg-country="CA" tvg-language="French" tvg-logo="https://i.imgur.com/byTTNAi.png" group-title="General",ICI Tele (720p)
-https://rcavlive.akamaized.net/hls/live/696615/xcancbft/master.m3u8
```

Comment:
This link works fine
![image](https://user-images.githubusercontent.com/7253922/134680793-2e3ee6d4-9521-49e6-a737-6d814d41c258.png)
and the new one only works through a VPN

So it would be better to leave both links in the playlist and mark the new one as `[Geo-blocked]`.

Reply:
Makes sense, could not test new link using streamtest.in (reported everything as offline). Fixed commit.